cogs/fun.py

```diff
@@ -202,25 +203,34 @@ async def wolfram(self, ctx, *, query):
 			except asyncio.TimeoutError:
 				raise QUERY_ERROR
 
+			log.debug(resp.url)
+
 			j = loads(j)
 			res = j['queryresult']
 
-			success = res['success']
+			success = bool(res['success'] and res['numpods'])
```

Comment:
Wrapping in bool(...) is not necessary since `x and y` returns a boolean value already